backend/people/migrations/0018_update_holding_team_orders.py

```diff
@@ -0,0 +1,52 @@
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    WSS = apps.get_model('WSS', 'WSS')
+    HoldingTeam = apps.get_model('people', 'HoldingTeam')
+    Staff = apps.get_model('people', 'Staff')
+
+    wss = WSS.objects.get(year=2020)
+
+    ordered_holding_teams = [
+        'Content',
+        'Technical',
+        'Network',
+        'Branding',
+        'Social',
+        'Media',
+        'Presentation Management'
+    ]
+
+    HoldingTeam.objects.get(wss=wss, name='Event Director', order=1).update(name='Manager & Director')
+
+    Staff.objects.create(
+        name='Shayan Oveis gharan',
+        picture='media/2020/staff/Shayan-Oveis-gharan.jpg'
+    )
+
```

Comment:
Set `polymorphic_ctype` field for this object.

Reply:
Done